azure-pipelines.yml

```diff
@@ -30,15 +30,6 @@ jobs:
     configuration: Release
     queueName: Build.Windows.Amd64.VS2022.Pre.Open
 
-- template: eng/pipelines/test-windows-job.yml
```

Comment:
I was surprised to see these jobs removed entirely. I thought we could adjust the configuration to ensure they only run on rolling builds and not on PRs. Was I wrong on that?

Reply:
I do not think that is possible. @JoeRobich is looking into some stuff but are far as I can tell that is well beyond what azdo supports. Furthermore, I see no reason to keep these around. Someone needs to come to me with a real case where a bug only reproduces if all 3 variables in our test matrix are just so.

Reply:
Simply include the template only if `eq($(Build.Reason), "PullRequest")`. https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml